rust/01-hello-world/README.md

```diff
@@ -28,18 +28,23 @@ Add the target:
 rustup target add x86_64-unknown-linux-musl
 ```
 
-Put this into your .cargo/config:
+Build:
+
 ```
-[target.x86_64-unknown-linux-musl]
-linker = "x86_64-linux-musl-gcc"
+TARGET_CC=x86_64-linux-musl-gcc \
+rustc --target=x86_64-unknown-linux-musl \
+-C linker=x86_64-linux-musl-gcc main.rs
 ```
 
-Build:
+To run:
 
 ```
-TARGET_CC=x86_64-linux-musl-gcc \
-RUSTFLAGS="-C linker=x86_64-linux-musl-gcc" \
-cargo build --target=x86_64-unknown-linux-musl
```

Comment:
It could be helpful to have `cargo`-based example(s) as well, though that would need some more changes (:one: adding a `Cargo.toml`, :two: moving `main.rs` to `src/main.rs`, :three: specifying the path to compiled executable in `target/...`), so I just made this one match the "dynamic" steps.

Reply:
so this readme even though it isn't super explicit is mostly about working on mac with rust - linux is super straight-forward - it might be best to split these up, we do have a cargo example here: https://github.com/nanovms/ops-examples/tree/master/rust/03-rust-sqlx - or at least highlight linux vs. mac better than it is now

Reply:
That makes sense! (mac is my current dev environment also).